### PR TITLE
New ocsa.hackclub.com website, and google workspace set up

### PR DESCRIPTION
**Pull request by BrightTheBackpack**
We made a new website and are setting up google workspace

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1656,9 +1656,30 @@ nust:
     type: CNAME
     value: nusthackclub.netlify.app.
 ocsa:
-  - ttl: 1
+  - ttl: 3600
     type: TXT
-    value: google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+    values: 
+    -google-site-verification=tk_snQuv19iUwoOlsbKZv2Wwg5stfcusEm1Q8NxDiVA
+    -v=spf1 include:_spf.google.com ~all
```
Comment (JosiasAurel) [inline hackclub.com.yaml:1663]: should it be `- v=spf1`? and also `- google-site-...`?

Comment (JosiasAurel) [reply hackclub.com.yaml:1663]: i think there should be space between the `-` and the value

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (BrightTheBackpack) [reply hackclub.com.yaml]: done!

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1656,9 +1656,30 @@ nust:
...
+      - exchange: alt1.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt2.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt3.aspmx.l.google.com.
+        preference: 10
+      - exchange: alt4.aspmx.l.google.com.
+        preference: 10
+  - ttl: 1
+    type: ALIAS
```
Comment (Muirrum) [inline hackclub.com.yaml:1679]: Should this be an ALIAS record? GitHub Pages wants a CNAME (https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#dns-records-for-your-custom-domain)

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 2 above)
Comment (BrightTheBackpack) [reply hackclub.com.yaml:1679]: idk, on the readme it says if your using google workspace change it to ALIAS. Also when it's on cname theres an error on the octodns test

Comment (Muirrum) [reply hackclub.com.yaml:1679]: Ah yeah, that's my bad.

**Review by Muirrum — APPROVED** [review]
lgtm

**Review by JosiasAurel — APPROVED** [review]

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1662,9 +1662,30 @@ nust:
...
+    value: uhegg.github.io.
```
Comment (Muirrum) [inline hackclub.com.yaml:1686]: Is this the correct GitHub pages URL? There doesn't seem to be a repo at uhegg/uhegg.github.io nor is there a site at uhegg.github.io

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 3 above)
Comment (BrightTheBackpack) [reply hackclub.com.yaml:1686]: the websites here: https://uhegg.github.io/hackclub, but i beleive github doesn't want you to include anything after the slash

**Review by Muirrum — APPROVED** [review]

**Pull request merged**